crypto/rand/rand_win.c

```diff
@@ -95,41 +87,4 @@ void RAND_screen(void)
 }
 #endif
 
-/* feed timing information to the PRNG */
-static void readtimer(void)
```

Comment:
High speed time is a decent source: a few of the low order bits (8-10) anyway.  What about calling `OPENSSL_rdtsc` and mixing that in too?  I'd do this in `RAND_poll` above rather than having a separate function.